<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newCourse</class>
 <widget class="QDialog" name="newCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>511</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>511</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 14, 43)</string>
  </property>
  <widget class="QLabel" name="insName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Instructor Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="couseName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Course Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hours">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Hours</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="maxstd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Maximum Students</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Elective</string>
   </property>
  </widget>
  <widget class="QComboBox" name="preReq">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <item>
    <property name="text">
     <string>Pre Required Courses</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/i18n/Qt_en_GB.qm</selectedon>
     </iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>math1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>math2</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>113</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
